### PR TITLE
Add compat data for DedicatedWorkerGlobalScope (Worker API)

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/DedicatedWorkerGlobalScope

### CODE REVIEW

File: api/DedicatedWorkerGlobalScope.json [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          }
+        }
+      },
+      "console": {
```
Comment (Elchi3) [inline api/DedicatedWorkerGlobalScope.json]: Looks like this is actually on WorkerGlobalScope: https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/console
Let's remove this from this table.

File: api/DedicatedWorkerGlobalScope.json [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          }
+        }
+      },
+      "performance": {
```
Comment (Elchi3) [inline api/DedicatedWorkerGlobalScope.json]: Looks like this is actually on WorkerGlobalScope: https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/performance
Let's remove this from this table.

File: api/DedicatedWorkerGlobalScope.json [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+            "version_added": "5.1"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "ononline_onoffline": {
```
Comment (Elchi3) [inline api/DedicatedWorkerGlobalScope.json]: Looks like this is actually on WorkerGlobalScope:
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/ononline
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/onoffline

Let's remove it from this table.

File: api/DedicatedWorkerGlobalScope.json [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DedicatedWorkerGlobalScope/close",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DedicatedWorkerGlobalScope.json]: `true` for all 3 flavors of Chrome.

File: api/DedicatedWorkerGlobalScope.json [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+              "version_added": "54"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DedicatedWorkerGlobalScope.json]: `true` for both flavors of Opera.

File: api/DedicatedWorkerGlobalScope.json [hunk 6]
```diff
@@ -0,0 +1,332 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "postMessage": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DedicatedWorkerGlobalScope/postMessage",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DedicatedWorkerGlobalScope.json]: `true`

File: api/DedicatedWorkerGlobalScope.json (hunk 1 above)
Comment (maboa) [reply api/DedicatedWorkerGlobalScope.json]: 👍

File: api/DedicatedWorkerGlobalScope.json (hunk 2 above)
Comment (maboa) [reply api/DedicatedWorkerGlobalScope.json]: 👍

File: api/DedicatedWorkerGlobalScope.json (hunk 3 above)
Comment (maboa) [reply api/DedicatedWorkerGlobalScope.json]: 👍

File: api/DedicatedWorkerGlobalScope.json (hunk 4 above)
Comment (maboa) [reply api/DedicatedWorkerGlobalScope.json]: 👍

File: api/DedicatedWorkerGlobalScope.json (hunk 5 above)
Comment (maboa) [reply api/DedicatedWorkerGlobalScope.json]: 👍

File: api/DedicatedWorkerGlobalScope.json (hunk 6 above)
Comment (maboa) [reply api/DedicatedWorkerGlobalScope.json]: 👍

**Pull request merged**